on_the_porch/unified_chatbot.py

```
@@ -9,23 +9,24 @@
     sys.path.insert(0, str(_RAG_DIR))
 
 
 # Import RAG retrieval helpers; import SQL pipeline lazily only when needed
 import retrieval  # type: ignore  # noqa: E402
 
-# Local OpenAI client config (avoid importing app3 at module load)
+# Local Gemini client config (avoid importing app3 at module load)
 try:
-    from openai import OpenAI  # type: ignore
+    import google.generativeai as genai  # type: ignore
 except Exception:  # pragma: no cover
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)